en/reference/assertions.md

```diff
@@ -29,58 +32,612 @@ sign-key-sha3-384: <key id> # Encoded key id of signing key
+Qr6grtz/Ud1bR9ThR0KISTuRhE/2qFOuNCVTHU2lFYGrECF0vJYo42tW2R85ZdsOpdVngAMUAJOZ
+REZKML9cgnJqIDwuZ4DNl+684p71+mNeQtbg2608F8fiK2jFJa8mHY5kxhzFGrIPVXEVmPTRlc0Q
+dPdViCOeg4jOuCybKQ/DrzE7Vjqhgl6UbKX2JtLbSxwBuB65YijCTVbqZD+u58ek9N9Z6ZKrJz2o
+HXq6RZqTpkCagV3FIxRRGhBDWQCePY4tRNEj9+3u31Af9daswRlmrFXwWDf46KPKx+fJ9jPECknO
+s+jQ5ij3fpMfWOzceHHLhIDm9Wj7sypD+63v4KDaXzQ+8dM/acMraNAJHRvCOr7bvFz9j7OEqr7y
+/6/QMZdkeYO94e1OPB5e+Dya95oCVqJhf05BAIKOn933EjGT3Vnm7HT+EzCvLUbbjMPlJ6ZyNnp/
+lE+8N8EQWmdLC+OwJRNHgucjvcaR
+```
+
 ## system-user
 
 The system-user assertion is a permit by the brand for local system users to be created on its specified devices.
 
-    type:           system-user
-    authority-id:   account-id   // Owner of the key, must be the brand
-    brand-id:       account-id   // Assertion will only work on models of this brand
-    email:          string       // Email of user
-    series:         list         // List of series which should accept this assertion
-    models:         list         // Models which should accept this assertion
-    name:           string       // Optional person’s name, for context and for gecos
-    username:       string       // Local system username for the user
-    password:       string       // Password for local system user, encoded and salted with
-                                    an algorithm hard to brute-force ($6$rounds=...$...)
-    ssh-keys:       list         // SSH keys to authorize for connection
-    since:          utc-datetime
-    until:          utc-datetime // Required!
-    revision:       integer
+The format is as follows:
+
+```text
+type:           system-user
+authority-id:   <authority account-id>
+revision:       <int>
+brand-id:       <account-id>
+email:          <user e-mail>
+series:         <list of series which should accept this assertion>
+models:         <models which should accept this assertion>
+name:           <optional person’s name>
+username:       <system user name>
+password:       <encoded password>
+ssh-keys:       <list>
+since:          <UTC datetime>
+until:          <UTC datetime>
+sign-key-sha3-384: <key id> # Encoded key id of signing key
+
+<signature>                 # Encoded signature
+```
+
+The index is the tuple <`brand-id`, `email`>. `series` is as specified by the
+model assertion. These assertions must be signed by the brand.
+
+The `password` header must be encoded and salted. The `until` header is in this
+case required.
 
-    signature       authority-sig
+The simple addition of such assertions to a device assertion database should not
+be enough to trigger the user creation. This must be initiated explicitly (via
+`snap create-user`, or in the context of the auto-import mechanism for
+assertions from removable devices, which requires physical access to the device).
 
+This would be an example of this assertion:
+
+```text
+type: system-user
+authority-id: 4BKZlf4WMNBKgQfij0rftmp5BzDdVhlf
```

Comment:
This is my personal ID so it would be good if we don't put that into the documentation.